docs/griptape-framework/drivers/web-scraper-drivers.md

```diff
@@ -6,6 +6,33 @@ Web Scraper Drivers can be used to scrape text from the web. They are used by [W
 
 ## Web Scraper Drivers
 
+### Proxy
+
+The [ProxyWebScraperDriver](../../reference/griptape/drivers/web_scraper/proxy_web_scraper_driver.md) uses the `requests` library with a provided set of proxies to do web scraping. Paid webscraping services like [ZenRows](https://www.zenrows.com/) or [ScraperAPI](https://www.scraperapi.com/) offer a way to use their API via a set of proxies passed to `requests.get()`
+
+Example using `ProxyWebScraperDriver` directly:
+
+```python
+import os
+from griptape.drivers import ProxyWebScraperDriver
+
+query_params = [
+    "markdown_response=true",
+    "js_render=true",
+    "premium_proxy=true",
+]
+proxy_url = f'http://{os.environ["ZENROWS_API_KEY"]}:{"&".join(query_params)}@proxy.zenrows.com:8001'
```

Comment:
Where did you get this example url from?

From [the docs I found](https://docs.zenrows.com/features/proxy-ip#premium-proxies), I'd expect this to look something like `https://api.zenrows.com/v1/?apikey=YOUR_ZENROWS_API_KEY&url=https%3A%2F%2Fhttpbin.io%2Fanything&premium_proxy=true` (from the `cURL` example)

Reply:
this is from the example on the dashboard when you create a free account. i dont see it in the docs though. i figured this would be an easier generic implementation because the actual url to be scraped does not have to change.

Reply:
![Screenshot 2024-06-21 at 9 59 57 AM](https://github.com/griptape-ai/griptape/assets/43043466/944c1c1e-1b1a-424d-89f5-4153d7386839)